File: FetchData/student.php
<?php
// student.php

// Set the content type to JSON
header('Content-Type: application/json');

// Get the JSON data from the request
$data = json_decode(file_get_contents('php://input'), true);

// Check if the data is valid
if (isset($data['name']) && isset($data['email']) && isset($data['phone'])) {
    $name = $data['name'];
    $email = $data['email'];
    $phone = $data['phone'];

    // Here you would normally insert the data into your database
    // For demonstration, we'll just respond with success

    // Respond with success
    echo json_encode(['status' => 'success', 'message' => 'Student inserted successfully']);
} else {
    // Respond with an error if data is missing
    http_response_code(400); // Bad Request
    echo json_encode(['status' => 'error', 'message' => 'Invalid input']);
}
?>

File: FetchData/Student.java
package com.example.fetchdata;

public class Student {
    private String name;
    private String email;
    private String phone;

    public Student(String name, String email, String phone) {
        this.name = name;
        this.email = email;
        this.phone = phone;
    }

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }
    public String getPhone(){
        return phone;
    }

    // Getters and setters
}